libfwupdplugin/fu-bluez-device.c

```diff
@@ -0,0 +1,548 @@
+/*
+ * Copyright (C) 2021 Ricardo Cañuelo <ricardo.canuelo@collabora.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#define G_LOG_DOMAIN				"FuBluezDevice"
+
+#include "config.h"
+
+#include <string.h>
+
+#include "fu-bluez-device.h"
+#include "fu-common.h"
+#include "fu-device-private.h"
+#include "fu-firmware-common.h"
+
+#define DEFAULT_PROXY_TIMEOUT	5000
+
+/**
+ * SECTION:fu-bluez-device
+ * @short_description: a Bluez Bluetooth LE device
+ *
+ * An object that represents a Bluez Bluetooth LE device.
+ *
+ * See also: #FuBluezDevice
+ */
```

Comment:
Is there any reason this is BLE only?  I would think bluez will provide both LE and full energy devices.

Reply:
>that this is a BLE device and that uses Bluez as a backend.

Yes, I think it's a bit confusing otherwise.

Reply:
I think we can just `s/LE/`